src/Button.js

```diff
@@ -205,7 +209,7 @@ function Button(props) {
     buttonStyles.backgroundColor = "transparent";
   }
 
-  const ButtonType = highlight
+  const touch = highlight
```

Comment:
**touch** can be named **Touchable** or **ButtonInstance** and instead of **RenderButton** you can call it like this:
```js
<Touchable
  {...extraProps}
  disabled={disabled}
  activeOpacity={opacity}
  style={buttonStyles}>
  {children}
</Touchable>
```